### PR TITLE
Finn 2

### PR DESCRIPTION
**Pull request by fbuggy**

### CODE REVIEW

**Review by mamc3334 — CHANGES_REQUESTED** [review]
Needs changes

**Review by mamc3334 — CHANGES_REQUESTED** [review]
Merge in main and repush.

File: common/utility.hpp [hunk 1]
```diff
@@ -24,14 +25,22 @@ struct Image_Attributes {
 struct smallColor{
     uint8_t r, g, b;
     bool operator==(const smallColor &other) const {
+<<<<<<< HEAD
+        return (r == other.r && g == other.g && b == other.b);
```
Comment (mamc3334) [inline common/utility.hpp:29]: resolve merge conflict

File: common/utility.hpp [hunk 2]
```diff
@@ -24,14 +25,22 @@ struct Image_Attributes {
...
+>>>>>>> origin/main
     }
 };
 
 struct bigColor{
     uint16_t r, g, b;
     bool operator==(const bigColor &other) const {
+<<<<<<< HEAD
+        return (r == other.r && g == other.g && b == other.b);
+=======
```
Comment (mamc3334) [inline common/utility.hpp:41]: resolve merge conflict

File: imgaos/CMakeLists.txt [hunk 3]
```diff
@@ -6,4 +6,9 @@ add_library(imgaos
         aosinfrequentcolor.cpp
 )
 # Use this line only if you have dependencies from this library to GSL
-target_link_libraries (imgaos PRIVATE common Microsoft.GSL::GSL)
\ No newline at end of file
+target_link_libraries (imgaos PRIVATE common Microsoft.GSL::GSL)
+
+target_include_directories(imgaos
+        PUBLIC
```
Comment (mamc3334) [inline imgaos/CMakeLists.txt:12]: what does this do?

File: imgaos/CMakeLists.txt [hunk 4]
```diff
@@ -6,4 +6,9 @@ add_library(imgaos
...
+        ${CMAKE_CURRENT_SOURCE_DIR}
```
Comment (mamc3334) [inline imgaos/CMakeLists.txt:13]: What does this target_include ... do?

**Review by mamc3334 — APPROVED** [review]

**Review by 100496948 — APPROVED** [review]

**Pull request merged**